src/main/java/lotto/domain/LottoGenerator.java

```diff
@@ -0,0 +1,34 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoGenerator {
+
+    private static final List<Integer> LOTTO_NUMBERS = IntStream.range(1, 46)
```

Comment:
- `IntStream`의 `rangeClosed`를 사용해보시는 것도 좋을 것 같아요.
- 숫자 리터럴은 상수로 선언하여 의미를 부여해보면 어떨까요?
```suggestion
    private static final List<Integer> LOTTO_NUMBERS = IntStream.rangeClosed(MIN_LOTTO_NUMBER, MAX_LOTTO_NUMBER)
```